service/src/test/java/greencity/service/UBSManagementServiceImplTest.java

```diff
@@ -168,6 +170,26 @@ void checkPaymentNotFound() {
         Assertions.assertThrows(UnexistingOrderException.class, () -> {
             ubsManagementService.getOrderDetailStatus(100L);
         });
+    }
+
+    @Test
+    void deleteViolationFromOrderResponsesNotFoundWhenNoViolationInOrder() {
+        Violation violation = ModelUtils.getViolation();
```

Comment:
If you want to test that Exception was thrown you do not have to use if block here. Verify in the when state that return will be empty Optional like

Reply:
Thank you for your feedback